### PR TITLE
CB2-6921: enable updateEuVehicleCategory to handle vehicles with vin changes

### PR DESCRIPTION
**Pull request by Ian-Hodges**
## Description
Enable updateEuVehicleCategory to handle vehicles with vin changes
[CB2-6921](https://dvsa.atlassian.net/browse/CB2-6921)

### CODE REVIEW

**Review by tom-bjss — APPROVED** [review]

File: jest.config.js [hunk 1]
```diff
@@ -14,5 +16,9 @@ module.exports = {
     transform: {
         '^.+\\.tsx?$': 'ts-jest'
     },
-    testMatch: ['**/*.*Test.ts']
+    testMatch: ['**/*.unitTest.ts'],
```
Comment (gjulien-bjss) [inline jest.config.js:19]: Mostly curious why the change to just `**/*.unitTest.ts` ?

Comment (Ian-Hodges) [reply jest.config.js:19]: So it only includes the unit tests in the test explorer.
Before:
image
After:
image
